src/word/word.rs

```diff
@@ -0,0 +1,206 @@
+//! Generic Integer traits.
+
```

Comment:
Ah crap. Did I removed this trait from bitintr and that's why you had to recreate it?

Reply:
Yeah, it was mostly adding loads of trait bounds for the stuff from bitintr that it needed.